scripts/codegen/CAN/App_CanMsgs.dbc

```diff
@@ -588,6 +588,7 @@ BA_DEF_DEF_  "GenSigStartValue" 0;
 BA_ "BusType" "CAN";
 
 BA_ "GenSigStartValue" SG_ 507 Start_Switch 1; 
+BA_ "GenSigStartValue" SG_ 82 Direction_Command_INVR 1; 
```

Comment:
would be nice to put these in order (82 before 507)

Reply:
actually how does this actually work? does it automatically set the value in the can table to 1?

Reply:
> actually how does this work? does it automatically set the value in the can table to 1?

Yep

Reply:
> would be nice to put these in order (82 before 507)

resolved